distribution/lib/Standard/Base/0.0.0-dev/src/Network/HTTP/Response.enso

```diff
@@ -69,3 +121,19 @@ type Response
         type_pair = ["type", "Response"]
         cons_pair = ["constructor", "Value"]
         JS_Object.from_pairs [type_pair, cons_pair, ["headers", self.headers], ["body", self.body], ["code", self.code]]
+
+## PRIVATE
+type Unsupported_Content_Type
+    ## PRIVATE
+       A type representing an unsupported content type.
+
+       Arguments:
+       - content_type: The content type that is unsupported.
+    Value content_type
```

Comment:
What is the type here? From the `to_display_text` I conclude it is `Text | Nothing`. Can we specify it in the constructor? Makes the code easier to read.

Reply:
```suggestion
    Value (content_type : Text | Nothing)
```

Reply:
Shouldn't this be called error or problem?

```suggestion
    Error content_type
```

I see it is an error that is thrown around. The `Value` constructor usually is not used for those.

Reply:
```suggestion
    Error (content_type : Text | Nothing)
```